CondTools/L1TriggerExt/interface/L1ConfigOnlineProdBaseExt.h

```diff
@@ -156,9 +168,10 @@ bool L1ConfigOnlineProdBaseExt<TRcd, TData>::getObjectKey(const TRcd& record, st
 
   // If L1TriggerKeyExt is invalid, then all configuration objects are
   // already in ORCON.
-  edm::ESHandle<L1TriggerKeyExt> key;
+  // edm::ESHandle<L1TriggerKeyExt> key_token;
+  L1TriggerKeyExt key;
   try {
-    keyRcd.get(key);
+    key = keyRcd.get(key_token);
   } catch (l1t::DataAlreadyPresentException& ex) {
```

Comment:
What is this try-catch trying to achieve? A `Record::get()` is not going to throw `l1t::DataAlreadyPresentException`.

Reply:
This is a relic from previous code structure... I did just swap label with token. Unfortunately who wrote this left CMS so not really sure what was the point of the try/catch block. 
Are you suggesting changing the exception or removing the block?

Reply:
I was mostly curious about the aim of the try-catch, because from the framework point of view it does not do anything (except make the code harder to read, and make capturing the ESProduct by const reference impossible). Eventually it is up to L1T to decide what to do about it (possibly in a subsequent PR).